mmengine/runner/runner.py

```diff
@@ -687,6 +687,43 @@ def set_randomness(self,
             deterministic=deterministic,
             diff_rank_seed=diff_rank_seed)
 
+    def set_torch_cfg(self, torch_cfg: dict) -> None:
+        """Set torch variable to by config.
+
+        Args:
+            torch_cfg (dict): Contains key-value pair which defines the names
+                of torch variables and corresponding values.
+
+        Examples:
+            >>> torch_cfg = dict(
+            >>>    backends.cuda.matmul.allow_tf32=True,
+            >>>    torch.backends.cuda.matmul.allow_fp16_reduced_precision_reduction=True
+            >>>    )
+        """  # noqa: E501
+        for attributes, value in torch_cfg.items():
+            assert isinstance(attributes, str)
+            if attributes in [
+                    'backends.cudnn.benchmark', 'backends.cudnn.deterministic',
+                    'torch.backends.cudnn.benchmark',
+                    'torch.backends.cudnn.deterministic'
+            ]:
```

Comment:
I think it's better to remove all `torch.` prefix in `attributes` and then check if it's in the first 2 options. Otherwise some abnormal configurations like `torch.torch.backends.cudnn.benchmark` may spoil the thing.